src/riak_kv_pb_object.erl

```diff
@@ -190,8 +193,8 @@ process(#rpbputreq{bucket=B0, type=T, key=K, vclock=PbVC,
     case Result of
         consistent ->
             process(Req#rpbputreq{if_not_modified=undefined,
-                                  if_none_match=consistent},
-                    State);
+                                  if_none_match=undefined},
```

Comment:
Even though it's ignored, why not set `if_none_match` to `true` here, with a comment as to why you're setting it to true:

```erlang
%% HAY Even though this is ignored it's getting set to true here so you
%% know that later on in process() it's going to get set to true in the
%% options passed to put()
```

I'm just thinking about how to reduce confusion for the next person (if any) who looks at `if_none_match=undefined` and thinks WTF